Excelize demo (#32)
graalvm/graal-languages-demos: graalwasm/graalwasm-micronaut-excelice/README.md

# Excelize with GraalWasm Micronaut Demo

> This demo illustrates how GraalWasm can be used to embed Excelize , a WebAssembly Go library for creating and reading Microsoft Excel (xlsx) spreadsheets.

Reviewer: ```suggestion
This demo illustrates how GraalWasm can be used to embed [excelize-wasm](https://github.com/xuri/excelize-wasm/), a WebAssembly build of the Go [Excelize](https://github.com/xuri/excelize) library for reading and writing Microsoft Excel™ spreadsheets.
```

The demo also uses GraalJS to access the Excelize module through the WebAssembly JavaScript API.

## Preparation

> Install GraalVM for JDK 21 and set the value of `JAVA_HOME` accordingly.
> We recommend using [SDKMAN!](https://sdkman.io/). (For other download options, see [GraalVM Downloads](https://www.graalvm.org/downloads/).)
> 
> ```bash
> sdk install java 21-graal

Reviewer: ```suggestion
Install GraalVM for JDK 24 and set the value of `JAVA_HOME` accordingly.
We recommend using [SDKMAN!](https://sdkman.io/). (For other download options, see [GraalVM Downloads](https://www.graalvm.org/downloads/).)

```bash
sdk install java 24-graal
```
```suggestion
Install GraalVM for JDK 21 and set the value of `JAVA_HOME` accordingly.
We recommend using [SDKMAN!](https://sdkman.io/). (For other download options, see [GraalVM Downloads](https://www.graalvm.org/downloads/).)

```bash
sdk install java 21-graal
```

```

## Run the Application

To start the demo, simply run:

```bash
./mvnw package mn:run
```

When the demo runs, open the following URLs in a browser:

- http://localhost:8080/

> 

Reviewer: And then? Maybe say what the user can do in the browser.
